libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -1323,6 +1323,26 @@ void IRGeneratorForStatements::endVisit(FunctionCall const& _functionCall)
 		}
 		break;
 	}
+	case FunctionType::Kind::BytesConcat:
+	{
+		TypePointers argumentTypes;
+		vector<string> argumentVars;
+		for (ASTPointer<Expression const> const& argument: arguments)
+		{
+			argumentTypes.emplace_back(&type(*argument));
+			argumentVars += IRVariable(*argument).stackSlots();
+		}
+		m_code <<
+			"let " <<
+			IRVariable(_functionCall).part("mpos").name() <<
+			" := " <<
+			m_utils.bytesConcatFunction(argumentTypes) <<
+			"(" <<
+			joinHumanReadable(argumentVars) <<
+			")\n";
```

Comment:
```suggestion
		define(IRVariable(_functionCall)) <<
			m_utils.bytesConcatFunction(argumentTypes) <<
			"(" <<
			joinHumanReadable(argumentVars) <<
			")\n";
```
I think that's the same and was the idea of how to use `IRVariable` in cases like this - but doesn't really matter too much.